egs/madcat_ar/v1/local/tl/chain/run_cnn_e2eali.sh

```diff
@@ -0,0 +1,233 @@
+#!/bin/bash
+
+# e2eali_1b is the same as chainali_1a but uses the e2e chain model to get the
+# lattice alignments and to build a tree
+
+# local/chain/compare_wer.sh exp/chain/exp/chain/cnn_e2eali_1b
+# System                      cnn_e2eali_1b
+# WER                             10.78
```

Comment:
Are these numbers updated for text-localization?

Reply:
sorry, it is an old number, I will fix it.